types/voice-activity-detection/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "dtslint/dt.json"
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/voice-activity-detection/tslint.json;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.